Velaptor/Batching/IBatchingManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿// <copyright file="IBatchingManager.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace Velaptor.Batching;
+
+using OpenGL.Batching;
+
+/// <summary>
+/// Manages batch items that are used for rendering.
+/// </summary>
+internal interface IBatchingManager
+{
+    /// <summary>
+    /// Adds a texture item to the batch.
+    /// </summary>
+    /// <param name="item">The item to add.</param>
+    /// <param name="layer">The layer to add the item on.</param>
+    void AddTextureItem(TextureBatchItem item, int layer);
+
+    /// <summary>
+    /// Adds a font glyph item to the batch.
+    /// </summary>
+    /// <param name="item">The item to add.</param>
+    /// <param name="layer">The layer to add the item on.</param>
```

Comment:
Grammar.  Refactor to remove the "on" at the end of the sentence
```suggestion
    /// <param name="layer">The layer to add the item.</param>
```